target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh

```diff
@@ -267,6 +271,15 @@ platform_pre_upgrade() {
 	xiaomi,mi-router-wr30u-stock|\
 	xiaomi,redmi-router-ax6000-stock)
 		xiaomi_initial_setup
+		;;
+	arcadyan,wg620443)
+		if [ "$(rootfs_type)" = "tmpfs" ]; then
+			echo -e "/dev/mtd1\t0x0\t0x20000\t0x20000" > /etc/fw_env.config
```

Comment:
Please use `uboot-envtools` package for this instead.

https://github.com/openwrt/openwrt/blob/main/package/boot/uboot-envtools/files/mediatek_filogic